release-to-candidate/README.md

```diff
@@ -30,6 +30,7 @@ jobs:
 | `channel`                | The channel to release the snap to.                                                       |    N     | `latest/candidate` |
 | `launchpad-token`        | A token with permissions to create Launchpad remote builds.                               |    Y     |                    |
 | `repo-token`             | A token with privileges to create and push tags to the repository.                        |    Y     |
+| `show-build-log`         | Show the build log even if the build succeed                                              |    N     | `'false'`          |
```

Comment:
I'm not sure about this change, what's the use case for disabling the build log output?

In any case, I think if we do keep this, the default should be inverted.

Reply:
> what's the use case for disabling the build log output?

Is the build log necessary if the snap successfully? It just increases time taken by the workflow a little bit more. The current scenario shows the build log, even if  the build is successful.

Reply:
Yeh I think it is necessary - it doesn't add anything to build time (or at least, it's in the order of 10s of ms I suspect), and when a subsequent build fails it's useful to go back and compare. 

I think my personal preference is to revert this change

Reply:
Done :D